test/restClient.spec.js

```diff
@@ -0,0 +1,228 @@
+const expect = require('chai').expect;
```

Comment:
yes, you are right. I was testing it with an old node v.4.2.4
But now it is working also for node 4 ;)
great